tests/performance/test_data_analysis/statistical_data_analyzer.py

```diff
@@ -0,0 +1,274 @@
+
+        Returns:
+            dataframe (pd.Dataframe): Dataframe corresponding to the CSV.
+        """
+        dataframe = pd.read_csv(csv_path)
+        if len(dataframe) == 0:
+            raise ValueError(f"The file {csv_path} has not data rows or it has not CSV format")
+
+        return dataframe
+
+    def load_yaml_items(self, yaml_path: str) -> Tuple[str, List[str], Dict[str, Dict[str, float]]]:
+        """Process the YML file containing the elements to be analyzed during the test. In addition,
+        it obtains from this file the attributes of 'process_name', 'processes' and 'metrics'.
+
+        Args:
+            yaml_path (str): path to the YML file containing the elements to be analyzed.
+
+        Returns:
+            process_name (str): name of the process to be analyzed.
+            processes (list[str]): the different processes that are included in the 'process_name'.
+            metrics (Dict[str, Dict[str, float]]): metrics to be analyzed.
+        """
+        with open(yaml_path, 'r') as file:
+            config = yaml.safe_load(file)
+
+        processes_section = config.get('Processes', {})
+        process_name = list(processes_section.keys())[0]
+        processes = processes_section[process_name]
+        metrics = config.get('Metrics', {})
+
+        return process_name, processes, metrics
+
+    def print_dataframes_stats(self) -> str:
+        """Generate a PrettyTable with the statistics for each process and metric.
+
+        Returns:
+            output (str): String containing the comparative table.
+        """
+        output = ""
+
+        for process in self.processes:
+            baseline_data = self.baseline[self.baseline[self.process_name] == process]
+            datasource_data = self.datasource[self.datasource[self.process_name] == process]
+
+            for metric in self.metrics:
+                table = PrettyTable()
+                table.title = process + " - " + metric
+                table.field_names = ['Name', 'Mean', 'Max value', 'Min value', 'Standard deviation', 'Variance']
+                table.add_row([
+                    "Baseline",
+                    round(baseline_data[metric].mean(), 2),
+                    baseline_data[metric].max(), baseline_data[metric].min(),
+                    round(baseline_data[metric].std(), 2),
+                    round(baseline_data[metric].var(), 2)
+                ])
+                table.add_row([
+                    "Data source",
+                    round(datasource_data[metric].mean(), 2),
+                    datasource_data[metric].max(), datasource_data[metric].min(),
+                    round(datasource_data[metric].std(), 2),
+                    round(datasource_data[metric].var(), 2)
+                ])
+                output += table.get_string() + "\n\n"
+
+        return output
+
+
+class StatisticalComparator:
+    """Class that contains the necessary methods to perform a comparison between the statistics
```

Comment:
Same here, the first line should be a summary. Detailed information should be in a description section, separated by one line

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3